idseq_pipeline/commands/accessionid2seq_functions.py

```diff
@@ -151,9 +151,9 @@ def generate_alignment_viz_json(nt_file, nt_loc_db, db_type,
     result_dict = {}
     to_be_deleted = []
     error_count = 0
-    for accession_id, ad in groups.iteritems():
+    for accession_id, ad in groups.items():
```

Comment:
Maybe we should bundle the change from iteritems to items with the change from python2 to actually using python3? So we don't increase the memory overhead

Reply:
Hm ok I'll change this PR to just the print statements

Reply:
Sounds good. Another option could be to check if we're using python2 or python3 and choose iteritems / items as a consequence?